app/views/shared/_user.html.erb

```diff
@@ -0,0 +1,7 @@
+<li class="user">
+            <img src="<%= user.photo %>" class="image">
+            <div class="user-container">
+            <h1 class="user-name"><%= link_to user.name, user_path(user) %></h1>
+            <p class="number-posts">Number of posts: <%= user.posts.count %></p>
+            </div>
+            </li>
```

Comment:
- [ ] [OPTIONAL] It will be nice to properly indent these lines of code in order to have a better and clean code base.
you should have :
``` ruby
<ul class="users posts">
  <% posts.each do |post| %>
    <li class="user">
      <%= link_to post.title, user_post_path(@user, post) %>
      <p><%= post.text %></p>
      <p>Comments: <%= post.comments_counter %></p>
      <p>Likes: <%= post.likes_counter %></p>
    </li>
    
    <% if ren === 0 %>
      <% if post.comments.any? %>
        <%= render 'shared/comments', post: post %>
      <% else %>
        <h3>No hay comentarios</h3>
      <% end %>
    <% end %>
  <% end %>
</ul>
```

Reply:
@Whoistolu hi thanks for your review. DO you now how can i indent the .erb file I tried using Rubocop but doesn't work.

Reply:
Hello @sanieni6 , you could manually indent it just like I did above.

Reply:
well resolved! ✔️